unified_targets/configs/VIVAF4AIO.config

```diff
@@ -0,0 +1,97 @@
+# Betaflight / STM32F405 (S405) 4.0.0 Mar  7 2019 / 06:40:55 (fee4ee5e0) MSP API: 1.41
+
+board_name VIVAF4AIO
+manufacturer_id VIVA
```

Comment:
This is not a valid manufacturer id.

Reply:
Add it to the manufacturer id list? if so I will add it now.

Reply:
I tried to find an actual website where the company contact information and their products are listed, but Google came up empty...

Reply:
Yeah - I suspect that is the case until they have it available. They are a wholesaler. sales@vivafpv.com can be used to contact them directly - in the meantime TBS and a few other retailers/manufacturers will stock them (and I assume provide user support).

Reply:
Do you have any information on what name the company is trading under?